run-cnf-suites.sh

```diff
@@ -53,5 +53,18 @@ done
 
 FOCUS=${FOCUS%?}  # strip the trailing "|" from the concatenation
 
+if [ "$VERIFY_CNF_FEATURES" == "true" ] && [ "$TNF_MINIKUBE_ONLY" != "true" ]; then
```

Comment:
Will this work if running within our containerised version? I don't believe we install `docker` in the container (if that works) so I'd be surprised if it does.

Reply:
It will work if the docker socket is mounted. The first container will be able to start a container on the host. It's tricky with podman though. It's not an officially supported case and it would require --privilege.

Reply:
Ha, almost lost what I had in mind. I actually had a plan, just didn't recall when reading your comment ... 
Instead of exposing docker socket or running privileged podman, we should invoke cnf feature deploy container in the run-tnf-container script as well. Subsequently, the tnf container itself would always be invoked with "VERIFY_CNF_FEATURES" set to "true". In the future, the CNF specific tests in its own repo/container can be included the same way. The only reason why I didn't do it yet is because the github actions still use minikube anyways. When we have other use cases for running tnf container or the pipeline gets a real ocp lab, we would need this change for sure.